### PR TITLE
Feature/probinso/fusible phase dynamic flush

### PR DESCRIPTION
**Pull request by robinson96**

### CODE REVIEW

File: src/care/LoopFuser.h [hunk 1]
```diff
@@ -916,14 +922,14 @@ void LoopFuser<REGISTER_COUNT, XARGS...>::registerAction(const char * fileName,
             if (verbose) {
                printf("hit reserved flushActions\n");
             }
-            flushActions();
+            flush_now = true;
          }
          // if we are approaching the 2^31-1 limit proactively flush
```
Comment (liu15) [inline src/care/LoopFuser.h:927]: This comment is no longer appropriate.

**Review by liu15 — APPROVED** [review]

**Review by adayton1 — APPROVED** [review]
LGTM

File: src/care/LoopFuser.h (hunk 1 above)
Comment (adayton1) [reply src/care/LoopFuser.h:927]: @liu15, looks like you changed this comment on a different branch?

Comment (liu15) [reply src/care/LoopFuser.h:927]: Yeah, the other branch is a bugfix for this.

**Pull request merged**